cmd/ascii2der/scanner_test.go

```diff
@@ -224,6 +226,53 @@ indefinite long-form:2`,
 		},
 		true,
 	},
+	// Byte-string-like BIT STRING literals are parsed correctly.
+	{
+		"b``",
+		[]token{
+			{Kind: tokenBytes, Value: []byte{0x00}},
+			{Kind: tokenEOF},
+		},
+		true,
+	},
+	{
+		"b`1010101001010101`",
+		[]token{
+			{Kind: tokenBytes, Value: []byte{0x00, 0x55, 0xaa}},
+			{Kind: tokenEOF},
+		},
+		true,
+	},
+	{
+		"b`101010100101`",
+		[]token{
+			{Kind: tokenBytes, Value: []byte{0x04, 0x55, 0x0a}},
+			{Kind: tokenEOF},
+		},
+		true,
+	},
+	// We can stick a | in the middle of a BIT STRING to add "explicit" padding.
+	{
+		"b`101010100|1010101`",
+		[]token{
+			{Kind: tokenBytes, Value: []byte{0x07, 0x55, 0xaa}},
+			{Kind: tokenEOF},
+		},
+		true,
+	},
+	// "Implicit" padding is zeroed in spite of explicit padding.
```

Comment:
It took me a bit to figure out what this was saying. Perhaps:

  // If explicit padding does not end at a byte boundary, the remaining padding bits are zero.